### ISSUE TITLE
Предметы: Грибы и ягоды

### ISSUE THREAD

**Issue by FaceLess1** [uonline/universe#163]
наряду с остальными элементами (типа трав и прочего) в локациях типа "Лес" или что-то  подобное предлагаю ввод грибов и ягод. отличный вариант: можно употреблять так, сырыми, можно готовить (вариант -- отварить). 
**грибы** и **ягоды** делятся на 2 группы по своему непосредственному действию: нейтральные и ядовитые. ядовитые накладывают некий эффект вроде снижения определенного параметра (как правило -- здоровье), к непосредственному употреблению не рекомендуются.
грибы и ягоды могут входить в состав зелий, правильно приготовленное зелье может нейтрализовать эффект ядовитого гриба и обратить его действие на полезный эффект. 
можно предусмотреть, что некоторые зелья буду полезными только при предварительном употреблении того или иного ядовитого гриба или ягоды. 
часть ядовитых грибов можно приготовить (отварить) и уже употреблять. 
также можно предусмотреть, что некоторые ядовитые грибы для одной расы будут нейтральными для другой -- как вариант. но поскольку инопланетян мы не предусматриваем, а все игровые расы (хоть уже существующие хоть обговариваемые) так или иначе гуманоидные со сходной системой пищеварения и устройства органической химии в организме, то над необходимостью ввода такой сложности сомневаюсь. 

когда впервые персонаж найдет такой гриб -- наверное, можно дать игровую подсказку вроде "хм, выглядит не очень съедобно!". и действительно это будет ядовитый гриб. скрывать не вижу смысла -- первая же десятка наколовшихся (если скроем свойства) полезет в чатик и скажет: "не ешьте Грибожуй -- он ядовитый и отнимет половину хп!".

ягоды растут на кустарниках в лесу, грибы растут под деревьями и на деревьях. 

само использование в приготовлении зелья можно описывать в рецепте. рецепт должен четко предусматривать какой гриб/ягода, в каком виде и в каком порядке он должен быть добавлен в состав. 

**upd:** я планирую также сюда выкладывать те рисунки грибов, которые у меня будут появляться.

**Comment by FaceLess1** [comment]
сразу прошу обратить внимание на шляпки: мне кажется, что стоит растушевать. нет?
![epson069](https://cloud.githubusercontent.com/assets/4042329/5764940/be1a0192-9d06-11e4-8fef-befc5b1bb722.JPG)

**Comment by m1kc** [comment]
В смысле? Пальцем растереть? Ну я бы попробовал.

**Comment by 3bl3gamer** [comment]
А как быть с названиями? Всмысле, названия каких грибов/ягод/трав игроку известны?

Сразу все? Но это может быть не очень логично, особенно если какой-нибудь лесной эльф, попав в глубокие подземелья, скажет "О, плесень Адской Накипи! Может пригодиться."

Не сразу все (тут фраза "хм, выглядит не очень съедобно!" более подходит)? Но тогда какие? Некоторые базовые + все квестовые + собранные-изученные?

Аналогично и для других предметов. Особенно редких и штучных.

**Comment by FaceLess1** [comment]
нет, если пальцем растереть -- будет очень хуево. практика показывает, что лучше не трогать оригинальный рисунок пальцами. я выше выложил растушеванный вариант с помощью MyPaint

**Comment by FaceLess1** [comment]
это, кстати, хороший вопрос. надо обдумать. 
вообще, в #165 я предлагал в записках, письмах указывать на некие предметы мета их нахождения. вероятно я сделаю позже также стилизованный под скетч рисунок этого же самого гриба. 
все эти грибы будут иметь различное изображение. не будет ситуации с ними, например, когда меняется только цвет ножки или шляпки, а сама геометрия гриба/ягоды и композиция одинаковая на рисунке. потому по скетчу можно будет узнать тот или иной гриб. пример скину ниже.

**Comment by FaceLess1** [comment]
вот пример вида записки. чисто так, из головы за несколько минут взял текст и зарисовку.

![111223](https://cloud.githubusercontent.com/assets/4042329/5777750/7dfe6008-9da6-11e4-9d5a-84f01fe1a2f2.jpg)

**Comment by FaceLess1** [comment]
и да, я не планировал особо углубляться в биологию и начинать разбирать где там "низшие грибы", где там "высшие грибы", к какому отделу они принадлежат и прочее. 
я беру реально существующие виды только потому, чтобы не рисовать ахинею, чтобы результат не выглядел совсем по-идиотски. 
однако я не заморачиваюсь ни над названием существующего прототипа, ни его таксонометрическим отделом. 
я изначально презумирую, что в игре все грибы принадлежат к "высшим". однако наличие "низших" грибов может дать новые возможности. например, можно некоторую еду не юзать, а через время она заплесневеет -- и именно в таком виде она может существенно повысить действие зелья. ну, думаю, понятно. 
просто я не настолько крут, чтобы нарисовать две разные плесени. я и одну с трудом нарисую. а ведь известно, что низшие грибы довольно разнообразны, разные продукты могут поражаться разной плесенью. можно забить хер и предположить, что у нас всего одна плесень, которая поражает любые продукты, и именно сам продукт (с плесенью) имеет значение, а не интерес в виде плесени в зависимости от продукта. но это как-то все для меня туманно. я остановился на "высших грибах". благо, там есть где развернуться, я думаю грибов 8 нафигарить, чтобы в зельях можно было использовать максимум 3 вида в различных комбинациях -- поверьте, это даже при таком скудном ассортименте даст довольно большие возможности для экспериментов в зельеварении. у нас может столько зелий и не оказаться.

**Comment by FaceLess1** [comment]
предвижу вопрос о наличии таких неопознанных грибов в инвентаре. насчет их наличие пока что в голову приходит такой вариант: в инвентарь помещается предмет с названием **?**, **undefinied** или каким еще -- принципиальной разницы мне нет. сложность будет состоять в том, что таких неопознанных в инвентаре может быть несколько видов. и это только грибы. будут и другие материалы и предметы. потому, вероятно, вам придется параллельно с названием присваивать предмету и некий номер из сформированного в будущем нашего внутриигрового каталога предметов, название программно запретить отображать, если вещь не идентифицирована.
как только в руки попадает такая записка (соответствующая вещи в инвентаре), то гриб идентифицируется. 
стоит заметить, что в записке в данном случае находится только его название и обстоятельства (в том числе и место нахождения) приобретения этого гриба. 
мы можем предполагать, что был некий Анонимус, который оказался в тех обстоятельствах, отыскал этот гриб и потянул в рот потому, что он дико хотел жрать. он уже собирался отрезать себе член и сожрать, но тут попался гриб. в конкретной записке не указаны свойства гриба. может, и я не отрицаю такой возможности, случиться так, что про один и тот же предмет будет несколько записок. в какой-то записке есть название предмета, в другой -- описание его свойств. в третьей записке может находиться расширенный список локаций (хотя и, возможно, не исчерпывающий), где этот конкретный предмет можно найти. общее у всех записок будет одно: каждая записка содержит в себе скетчевое изображение предмета или его название. может ли появиться записка, в которой будет альтернативное название конкретного предмета? во имя избежания путаницы -- эту вероятность я убираю. или такое название может быть (новая записка с разоблачением, пример текста: "я слышал об Анонимусе, прозвавшим тот гриб "Сытным". должен сказать, что он знатный тролль. вы никогда не интересовались почему та записка -- была его последней? а я отвечу: тот гриб -- последнее, что он съел. он отбирает половину вашего максимального здоровья. поскольку Анонимус на момент поедания гриба был уже изрядно ослабшим -- он не смог бороться с действием гриба. поэтому я считаю, что стоило тот гриб назвать "Гриб Истощения". по крайней мере, это было бы честно"). 
однако у предмета в любом случае есть "главное" название для избежания путаницы в инвентарях, при торговле и прочее.

**Comment by FaceLess1** [comment]
иной способ "опознать гриб" -- заюзать некий предмет. либо мы остановимся на неком зелье Идентификации, в которое потребуется добавить ингридиент для опознания. потом это зелье выливается на пергамент, а на пергаменте начинают "проявляться" чернила, складывающиеся в текст. в тексте опишутся свойства. но это не даст информации о названии.
можно для опознания использовать некий свиток Идентификации -- как в Диабло. эту идею я давал уже давно, сейчас реально не могу быстро найти упоминание. эффект такой же, как у зелья. поэтому надо будет выбрать: свиток или зелье будем вводить в игру в случае чего.
можно тупо сожрать и оценить эффект. эффекта может и не быть -- необязательно должно быть +200 к урону прямо вот в самый ближайший после употребления момент. я пока не обдумывал -- я просто рисую прикольные грибы, не определился еще. в любом случае способ довольно небезопасный при любом раскладе.
ну и искать записки, выторговывать записки. 
если сейчас кто-то мне начнет говорить, мол, с маленькими предметами зелье -- гуд, а как меч опознать, то тоже все просто: меч держим над пергаментом (ну, представляем, что то так), зельем поливается тот несчастный меч, а с меча уже капает это зелье прямо на пергамент, где чернила вырисовываются в текст.

**Comment by FaceLess1** [comment]
однако, повторюсь: 
все это приведет к тому, что в чатике будет такое:
"найдете "Сытный гриб", но ни в коем случае не ешьте -- это наебка, вам сразу пиздец после того, как сожрете!!!"
"найти Сытный гриб можно в: "Дремучем лесу", "Изумрудной гавани" и "Пизде русалки", поищите там"
"Сытный гриб -- ингридиент из Зелья Энергии, таких нужно на него сразу два!"

то есть сам поиск для первых нескольких сотен игроков будет реальным челленджем, но с развитием -- появится нечто (мне было бы приятно) Вики по проекту на всяких там wikia.com и группках в соц.сетях (нет, реально мне бы это польстило). и там будет просто дохуища огромный склад инфы. быть может, даже инфа о багах, которые мы каким-то образом упустим. 
потому вся вот эта конспирация -- ненадолго.
хотя, Антон, ты безусловно прав: чисто логически эльф не может спуститься впервые в подземелье и узнать ту самую "Адскую плесень", если он вообще не знал никогда что это такое.

**Comment by m1kc** [comment]
> я выше выложил растушеванный вариант с помощью MyPaint

Второй выглядит хуже.

**Comment by m1kc** [comment]
Тут, наверное, будет ко мне вопрос, почему я про рисунок уже два сообщения оставил, а остальную стену текста не прокомментировал. Но я всё ещё обдумываю.

**Comment by FaceLess1** [comment]
хорошо, что мне изменить в рисунке? потому что "выглядит хуже" -- как бы слегка неопределенно.

**Comment by FaNatizM** [comment]
Шляпка выглядит неестественно по сравнению с остальным рисунком, дисбаланс, на мой взгляд. Кстати, у гриба на нижней стороне шляпки должны быть отверстия/щели-складки для спор, я б нарисовал...

**Comment by FaceLess1** [comment]
вопрос еще состоит в том, какое разрешение будет у рисунка и придется ли мне рисовать миниатюру для инвентаря. потому что при уменьшении такие детали вроде "отверстий" будут выглядеть херово из-за сжатия. или не будут видны вовсе. мне не лень потратить еще час, но целесообразно ли это?

**Comment by m1kc** [comment]
> что мне изменить

Ну, ничего не менять, например.

> какое разрешение будет у рисунка

Вот понятия не имею. Я намерен хуячить всё крупно вплоть до пол-экрана, потому что считаю, что рисунки создают атмосферу (и я очень страдаю от того, что никто по сей день не хочет иллюстрировать Кронт). С другой стороны, понятно, что в инвентаре пол-экрана ну никак не получится - там будут иконки. Какого размера - я хз. Зависит от того, как он будет организован и сколько в среднем различного говна будет в рюкзаке у среднего игрока. Возможно, с добавлением нового контента иконки будут мельчать или наоборот - усиливается специализация, разнообразия всё меньше. Не знаю, короче. Просто не парься пока на эту тему.

> у гриба на нижней стороне шляпки должны быть отверстия/щели-складки для спор

Да, вот, учись вниманию к мелочам. На мой взгляд, такие детали рука должна вырисовывать сама ещё до того, как мозги начнут думать о разрешении.

**Comment by FaceLess1** [comment]
я достаточно в теме о том, что на нижней стороне шляпки есть пазухи для спор грибов и пока что не настолько ослеп, чтобы на оригинале рисунка их не увидеть. однако я пока рисовать не стал по вполне уже объясненным причинам. никто же не нашел этот рисунок в папке репозитория? то-то же, я сначала выложил его на обсуждение.

**Comment by FaNatizM** [comment]
Кстати, а точно нужно будет рисовать миниатюру, мне казалось можно просто уменьшать разрешение картинки?

**Comment by m1kc** [comment]
Не, я нигде не говорил "рисовать миниатюру", я собираюсь уменьшать.

**Comment by FaceLess1** [comment]
вот потому, что "уменьшать" -- я и не стал рисовать отверстия для спор. практика мне подсказывает, что получается дикая херня, если на рисунке в изобилии прорисованы детали. поэтому я заранее смотрю на изначально необходимый размер рисунка, чтобы определяться с детализацией.

**Comment by m1kc** [comment]
В смысле? Ну проебутся детали и проебутся, делов-то.

**Comment by FaceLess1** [comment]
действительно, прорисовать детали, чтобы они потом проебались -- это же так... логично)
ок, в общем, я понял, посмотрю что могу сделать с деталями.

**Comment by 3bl3gamer** [comment]
> прорисовать детали, чтобы они потом проебались

Ну почему. Одно дело - смотреть миниатюру на обычном экране, другое - на какой-нибудь "Самсунг Мега Галактическая Лопата HD", где плотность пикселей больше раза в 3.

**Comment by FaceLess1** [comment]
я не спорю, что на iPad наиболее высокое количество пикселей на дюйм, а еще можно подключить мобилу к плазме на 42 дюйма и играть вот так.
однако мне помнится, что минимальное разрешение готовим 640х320, поэтому меня вот этот момент беспокоит. однако что-то это уже не в тему совсем.
я же сказал, что поработаю с деталями, мои вопросы в этом ключе закрыты.

**Comment by m1kc** [comment]
Кстати, да, ретина же. Я про неё периодически забываю, потому что у меня её нет, но мы её поддерживаем.

**Comment by FaceLess1** [comment]
появился более детализированный вариант.

**Comment by m1kc** [comment]
Да, так лучше.

**Comment by FaceLess1** [comment]
> старые варианты удалены. ну и да -- просто скинуть текущий вариант в папку уже?

**Comment by m1kc** [comment]
Да, давай.

**Comment by FaceLess1** [comment]
![epson072-1](https://cloud.githubusercontent.com/assets/4042329/5874492/5602add0-a313-11e4-8e83-d970d14f2b2a.jpg)

**Comment by FaceLess1** [comment]
еще вопрос: стоит ли их рисовать "парно" или оставить по одному грибу на рисунке? получается, что в инветаре 1 гриб, а нарисовано два.

**Comment by m1kc** [comment]
Ну хз, мне любое количество норм, вряд ли люди настолько тупые. Кроме того, это можно интерпретировать так, что эти грибы берутся только парами и расходуются тоже.

**Comment by FaceLess1** [comment]
я просто не хочу усложнять. с ягодами тогда как -- гроздьи?
(грибов будет больше, чем 8 видов, настроен серьезно)

**Comment by m1kc** [comment]
Ну да - гроздья, веточки. Не рисовать же одну ягоду, в конце-то концов.

**Comment by FaceLess1** [comment]
понятно. второй гриб кидать в папку?

**Comment by m1kc** [comment]
Эм... А что это за гриб-то вообще?

**Comment by FaceLess1** [comment]
что это за гриб и каковы его свойства в игре или что это за гриб в реальном материальном мире?
впрочем, неважно. свойства грибам я не придавал еще, а что это за гриб в реальном мире -- да мне похеру что это и как он назывался. мне доставил его вид. будет пара-тройка грибов, которые тебя, наверное, удивят своим видом.

**Comment by m1kc** [comment]
Ну я, типа, хотел с фотографией сравнить. Так я могу только сказать, что левому грибу стоит добавить объёма.

**Comment by FaceLess1** [comment]
![images 2](https://cloud.githubusercontent.com/assets/4042329/5879244/8602134c-a33e-11e4-8b19-3830f2957450.jpg)

вот оригинал.

**Comment by FaceLess1** [comment]
не в тему:  а где наша карточка с приоритетными целями?

**Comment by m1kc** [comment]
#74?

**Comment by FaceLess1** [comment]
да, спасибо, именно это я и искал.

**Comment by FaceLess1** [comment]
![epson072-2](https://cloud.githubusercontent.com/assets/4042329/5891584/b49c2be4-a4af-11e4-8d8f-5d04f1cebc23.jpg)
![epson074](https://cloud.githubusercontent.com/assets/4042329/5891585/b8e81816-a4af-11e4-8642-1de61c3d410a.JPG)

**Comment by m1kc** [comment]
Лайк.

**Comment by FaceLess1** [comment]
оба? старую версию удалять?

**Comment by m1kc** [comment]
Угу. Закинь в папку, если не тяжело.

**Comment by FaceLess1** [comment]
нарисую четвертый и скину сразу пак.

**Comment by FaceLess1** [comment]
![epson075-1](https://cloud.githubusercontent.com/assets/4042329/5893509/447a624c-a4f8-11e4-9e91-271b92c6230a.jpg)

**Comment by FaceLess1** [comment]
ну так чего, все 4 кидать?

**Comment by m1kc** [comment]
У четвёртого гриба вот эта фигня на ножке какая-то странная, на лист металла похожа.

**Comment by m1kc** [comment]
Остальные три можно.

**Comment by FaceLess1** [comment]
это -- часть чашечки. от металла там нет ничего.

**Comment by m1kc** [comment]
![2015-01-27 17 50 19](https://cloud.githubusercontent.com/assets/1831620/5920327/f8bed22e-a64c-11e4-93bb-ef78bffc2b01.png)

Вот эта.

**Comment by FaceLess1** [comment]
вот эта, да.

**Comment by m1kc** [comment]
Ну блин, свет на ней какой-то странный. С резкими переходами, чуть ли не бликами.

**Comment by FaceLess1** [comment]
![epson075-2](https://cloud.githubusercontent.com/assets/4042329/5920527/478beeb8-a64e-11e4-9963-e643163a4351.jpg)
а так?

**Comment by m1kc** [comment]
Э-э-э... Шило на мыло, по-моему. Было два больших резких перехода, стало двадцать маленьких.

Начнём с простого. Что это за гриб? Как выглядит оригинал? Откуда падает свет? В какой стороне должна быть тень? Из какого материала эта штука? Как на неё должен ложиться свет? Может ли она бликовать?

И, за компанию, дно ножки. Только что заметил, что оно белое. Почему?

**Comment by FaceLess1** [comment]
это шиитаке. дно ножки белое потому, что срез там белый, а в говне вымазать еще не успели.

![shiitake](https://cloud.githubusercontent.com/assets/4042329/5920679/691c3500-a64f-11e4-9140-126ec2cff302.jpg)

**Comment by m1kc** [comment]
Оригинал, не пересвеченный:

![shiitake](https://cloud.githubusercontent.com/assets/1831620/5920770/1b836be6-a650-11e4-80ab-720e78396e19.jpg)

Дно действительно гораздо светлее остальной ножки, но оно не белое.

Ещё хочется отметить, что тут эта фигня не имеет чёткого контура и совсем не похожа на лист металла, но, думаю, ты это и сам видишь.

**Comment by FaceLess1** [comment]
да, тут, на оригинале "блик" еще более резкий, чем у меня, зашибись просто.

**Comment by m1kc** [comment]
Где тут блик-то?

**Comment by FaceLess1** [comment]
я написал "блик", короче. спорить я не хочу и не буду. чего там, ножку новую?

**Comment by m1kc** [comment]
Если не лень - да. Если лень - можешь и первую версию залить.

**Comment by FaceLess1** [comment]
я елаю версию вообще без чашечки. минут 10 -- будет готово.

**Comment by FaceLess1** [comment]
оп, готово
![epson075-3](https://cloud.githubusercontent.com/assets/4042329/5921697/d3ef6f80-a656-11e4-871a-a9dd79b29c22.jpg)

**Comment by m1kc** [comment]
Ну и ок.

**Comment by FaceLess1** [comment]
залито.

**Comment by m1kc** [comment]
Окей, клёво. Особый лойс, наверное, вот этому:

![https://raw.githubusercontent.com/uonline/universe/c15af9fb04752801d6f258be83518ed4ddb6ae30/_InputStream/items/mushrooms/EPSON074.JPG](https://raw.githubusercontent.com/uonline/universe/c15af9fb04752801d6f258be83518ed4ddb6ae30/_InputStream/items/mushrooms/EPSON074.JPG)

**Comment by FaceLess1** [comment]
мне он тоже очень нравится. постараюсь следующие сделать в подобном качестве. хотя парочка грядет не совсем обычный в привычном понимании гриба.

**Comment by FaceLess1** [comment]
подводя итоги, обобщаю: 
грибы -- организмы, которые произрастают преимущественно в лесах и пещерах в зависимости от вида. по своему назначению могут применяться в пищевых и лекарственных целях, также в составе зелий могут давать особые эффекты. по своему действию делятся на пищевые и ядовитые. смешивание нескольких видов грибов в одном зелье не значит суммирование свойств, такое зелье может оказать неожиданный эффект. один гриб занимает в инвентаре 1 клеточку, то есть минимальный размер. описания, свойства и прочее о грибах можно выяснить из разговоров, писем и любым другим способом, который подвернется. также можно изготовить предмет "книга грибов".
ягоды -- произрастающие на деревьях и кустарниках плоды, пригодные или же непригодные в пищу. по своему действию также делятся на съедобные и ядовитые. также могут применяться в зельях, суммирование эффектов ягод в одном зелье не дает сумму их эффектов. одна гроздь ягод занимает одну клетку в инвентаре -- минимальный размер. можно изготовить предмет "книга ягод".

**Comment by m1kc** [comment]
Не вижу подписи @FaNatizM.

**Comment by m1kc** [comment]
Осталось перенести резолюция куда-то - в файлы или на вики - и карточку закрыть.

**Comment by FaNatizM** [comment]
Эту карточку я подписал. Напоминаю, что изначально условились подписываться "пальцами вверх" под результирующим постом. Вон он стоит 12 дней назад.

**Comment by m1kc** [comment]
Процитирую ещё раз последнее сообщение:

> Осталось перенести резолюцию куда-то - в файлы или на вики - и карточку закрыть.

Чтобы не было больше такого недопонимания, рассказываю ещё раз жизненный цикл карточки в рамках подготовки к релизу:

1. Карточка создаётся.
2. Карточка обсуждается.
3. Выносится резолюция.
4. Если все с ней согласны - ждём трёх подписей (@FaceLess1, @FaNatizM, @m1kc), в виде пальцев или словами, без разницы. Автор резолюции, естественно, с собой согласен, поэтому считается, что его подпись уже стоит и нужно ещё две.
5. Если кто-то не согласен - опять обсуждаем, выносим новую резолюцию, и так, пока подписи не соберутся.
6. Всё ценное из карточки выносится куда-то в актуальную рабочую документацию. В файлы universe либо вики. Чтобы понять, что считается ценным, представьте, что карточку после закрытия сожгут, а пепел закопают. Вот всё, что не хотелось бы терять, выносится. То есть это в первую очередь резолюция, готовые работы и что-то ещё по ситуации. И проставляются ссылки, куда что ушло.
7. Я вижу всё это и закрываю карточку.

**Comment by m1kc** [comment]
Так, а давайте-ка я попробую резолюцию вынести, а то тут в карточке столько интересного, что в предыдущей резолюции не отражено. Итак:

* Существует такой вид предметов - грибы.
* Грибы можно собирать. В основном в лесах и пещерах.
* Существует такой вид предметов - ягоды.
* Ягоды можно собирать с деревьев и кустарников.
* Грибы и ягоды можно есть. Каждый гриб или ягода может давать какой-то эффект, полезный или вредный, мгновенный или долговременный, немедленный или отложенный. _Кстати, насчёт отложенных не знаю - по-моему, бред выйдет. Но отказываться на этом этапе не будем._
* Грибы и ягоды лёгкие, занимают в инвентаре минимальный объём и стакаются до 20. _Всё это говорю наугад, потому что у нас до сих не утверждена концепция инвентаря, хотя какой-то инвентарь уже реализован._

* При подбирании гриба или ягоды его название и свойства неизвестны. Игра намекает, если гриб или ягода ядовиты.
* Существует такой вид предметов - зелье идентификации.
* Это зелье можно скрафтить - наверное, потому что крафта у нас пока нет, и примерный состав неясен - или купить у продавцов. Возможно, не у всех продавцов, а только определённых - алхимиков, например.
* Если смешать это зелье с грибом или ягодой, персонаж узнаёт его название. С этого момента он может опознать все грибы или ягоды такого вида.
* Существует такой вид предметов - записка.
* Записки можно найти на локациях или купить.
* Записки открывают одно или несколько свойств определённого гриба или ягоды. Записки не открывают название. Прочитав записку, персонаж запоминает эти свойства навсегда, и они с этого момента всегда упомянуты для него в описании этого гриба или ягоды. Игра даёт подсказку, все ли свойства гриба или ягоды известны персонажу.

* Есть такие виды предмета - книга грибов и книга ягод.
* Книгу грибов нельзя купить, а можно только изготовить. Изготовление дорогое и долгое. При прочтении книга открывает персонажу все названия и свойства всех или фиксированного набора грибов или ягод.

Подписываем как обычно.
